### DISCUSSION TITLE
RubyConf TH in 2022

### DISCUSSION THREAD

**Discussion by olivierobert** [bangkokrb/rubyconfth#170] [RFCs]
## Issue

Due to the COVID 19 pandemic, RubyConf Thailand has not been held since October 2019 😭 Since the organization team prefers to hold an in-person conference, the various restrictions did not allow to organize an event that would allow the participation of the maximum of people. However, the situation has now changed since Thailand dropped all restrictions as of June 2022.

As of now, the number of conferences planned for **Asia** for the remainder of the year is still fairly limited:

![image](https://user-images.githubusercontent.com/696529/176992204-1348bfc9-05e7-47f6-9f4b-5c2774cbfed9.png)

Even some events like Ruby Kaigi does not recommend foreigners to join in: https://rubykaigi.org/2022/covid-19/#non-japan-based-attendees

While Matz already confirmed his participation in RubyConf Thailand in October 2023 🙌, this is still far away in the future. And when interacting with local community members, we feel a desire to have more events to participate in. 

Eventually, the current economic environment (yes, that inflation) might make it harder to onboard the number of required sponsors for the event or have the participation of foreign attendees. 

## Solution

Since we have already passed the mid-point of the year, it does not leave much time for organizing an event 😅 However, there is definitely room to make it happen. 

First, we can cut down on organizational complexities by hosting the event at the same location as the event in 2019. [The venue](https://www.google.com/maps/place/Pullman+Bangkok+King+Power/@13.75893,100.5352453,17z/data=!3m1!4b1!4m5!3m4!1s0x30e29eb64a4f804b:0xe80009bce2fa812a!8m2!3d13.75893!4d100.537434?shorturl=1) was praised by speakers and attendees alike. 

Second, by deciding this month, we could have the CFP running for three months and still have two months to make the arrangements. This would also conicide with the best season of the year to visit Thailand 😎

Regarding sponsors, this time around, in addition to [Nimble](https://nimblehq.co/), we can also count the support of the Bangkok office of the Fukuoka Prefectural Goverrnment 🙌 . In addition, by running this RFC, we can test out the waters and try to gain early support before firmly committing to organizing the event. You (who read this RFC) can help us to share it to people you might think could be interested asbeing a sponsor for the event.

The plan:
- Venue: Pullman Bangkok King Power
- Date: last week of November or first week of December
- CFP: July 15-October 15 2022
 
## Who Benefits?

The local Ruby community but also the global Ruby community as we intend to have an international line-up of speakers and attendees.
 
## What's Next?

If we receive positive feedback, we would need to:

- [ ] Book the venue
- [ ] Prepare the sponsorship kit 
- [ ] Open the CFP

**Comment by matthewmayer** [comment]
I think it's feasible. Although we spent a year organising it last time the majority of the work was in the last couple months. 

My main concern is money, we did well getting sponsors last time but companies have been tightening their belts. I think we should try to sketch out a budget based on last times numbers. We'd have to make sure we have enough money in the bank early enough from sponsorship to cover the venue deposits as last time a lot of the ticket sales were quite last minute.

**Reply by matthewmayer** [reply]
Also I think given the short timescale we might not want to run a completely blind CFP like last time but instead reach out proactively to some people we think might want to talk and do a mix of handpicked talks as well as speakers from the CFP? Then we could also start announcing locked in speakers earlier to drive ticket sales.

**Reply by matthewmayer** [reply]
I do like the idea of basically cloning what we did last time in terms of venue, schedule etc, to avoid having to make too many decisions from scratch, with a few tweaks (maybe 4-5 fewer talks overall to make it less crammed)

**Reply by citizen428** [reply]
> Also I think given the short timescale we might not want to run a completely blind CFP like last time but instead reach out proactively to some people we think might want to talk and do a mix of handpicked talks as well as speakers from the CFP?

In all fairness, that is already mostlt what we did last time since a CfP alone is not sufficient to guarantee diversity in speakers. But we could lean even heavier into my/others connections this year.

**Comment by citizen428** [comment]
I'm in and would love to take care of the CfP and speakers in general again.

**Reply by olivierobert** [reply]
We were definitely hoping that you would be in. You did great with the speakers' lineup previously and it would be great to have you again. This time, let's have fewer speakers and more time for attendees to mingle 🤗

**Reply by citizen428** [reply]
Sure, once we know whether we'll go ahead with this or not we can plan out the number of talks.

**Comment by matthewmayer** [comment]
do you want to have a meet/zoom call some evening next week we could try to sketch out a timetable and solve any red flags?

**Reply by citizen428** [reply]
Sure, let’s coordinate via Slack.

**Reply by olivierobert** [reply]
Sure 👍 FYI I contacted the hotel venue to check for availability and rates per head per day 🤑

**Comment by keithrbennett** [comment]
To reinforce the point of the lack of Ruby conferences in Asia, the India conference is only 1 day. So if we have a full length conference it might be the _only_ one in Asia this year, other than Japan.
